Core/Core/View/Base/Webview/Models/WebviewInjection.swift

```diff
@@ -47,6 +47,11 @@ public extension WebviewInjection {
             .webviewInjection()
     }
 
+    static var inversionCss: WebviewInjection {
```

Comment:
@forgotvas Although the name is fine, but what do you think about renaming it to colorInversionCss, for more clear name as inversion can be of any type thoughts?

Reply:
renamed